### PR TITLE
Enables pylint w0631 w0311

### PR DESCRIPTION
**Pull request by cacarrara**
This change enables pylint check w0631. Since the only occurrence is on
external spark library, it was disabled specifically. It also enables w0311
(bad identation).

Signed-off-by: Caio Carrara ccarrara@redhat.com

### CODE REVIEW

File: selftests/checkall [hunk 1]
```diff
@@ -169,14 +169,13 @@ results_dir_content() {
...
 LINT_CMD="inspekt lint --exclude=.git"
-PYLINT_ENABLE="--enable R0401,W0101,W0102,W0104,W0105,W0106,W0107,W0108,W0109,W0111,W0120,W0122,W0123,W0124,W0125,W0150,W0199,W0211,W0222,W0232,W0233,W0301,W0312,W0401,W0404,W0406,W0410,W0601,W0602,W0603,W0604,W0611,W0612,W0613,W0614,W0622,W0623,W0631,W0640,W0711,W1201,W1202,W1300,W1301,W1302,W1303,W1304,W1305,W1306,W1307,W1401,W1402,W1501,W1503,W1645"
+PYLINT_ENABLE="--enable R0401,W0101,W0102,W0104,W0105,W0106,W0107,W0108,W0109,W0111,W0120,W0122,W0123,W0124,W0125,W0150,W0199,W0211,W0222,W0232,W0233,W0301,W0311,W0312,W0401,W0404,W0406,W0410,W0601,W0602,W0603,W0604,W0611,W0612,W0613,W0614,W0622,W0623,W0631,W0640,W0711,W1201,W1202,W1300,W1301,W1302,W1303,W1304,W1305,W1306,W1307,W1401,W1402,W1501,W1503,W1645"
 if [ "$AVOCADO_PARALLEL_LINT" ]; then
     LINT_CMD="$LINT_CMD --parallel=$AVOCADO_PARALLEL_LINT"
 fi
 
 run_rc lint "$LINT_CMD $PYLINT_ENABLE"
 
-# Skip checking test_utils_cpu.py due to inspektor bug
```
Comment (clebergnu) [inline selftests/checkall]: I wonder the reason for the removal of this comment.  The original point for having it was the assumption that anyone who would look at this, would immediately ask "what's wrong with test_utils_cpu.py?", and the cause for the exclude is really in `inspektor`.  If anything else, maybe this should be in a different commit.

Comment (cacarrara) [reply selftests/checkall]: Oops, it was not intentional. I'm going to add it back.
